Source/WebCore/platform/mediastream/libwebrtc/gstreamer/GStreamerVideoDecoderFactory.cpp

```diff
@@ -206,13 +205,22 @@ class GStreamerWebRTCVideoDecoder : public webrtc::VideoDecoder {
             return WEBRTC_VIDEO_CODEC_UNINITIALIZED;
         }
 
+        if (inputImage._encodedWidth && inputImage._encodedHeight)
+            updateCapsFromImageSize(inputImage._encodedWidth, inputImage._encodedHeight);
+
+        RELEASE_ASSERT(m_caps);
+        if (UNLIKELY(!m_caps)) {
```

Comment:
Isn't this redundant? Did you mean to just assert here? If you just want to assert here, you can just do `ASSERT_NOT_REACHED` before returning inside the if